go/consensus/tendermint/roothash/roothash.go

```diff
@@ -418,10 +419,11 @@ func (sc *serviceClient) reindexBlocks(currentHeight int64, bh api.BlockHistory)
 		}
 
 		// Index block.
-		tmEvents := append(results.BeginBlockEvents, results.EndBlockEvents...)
+		tmEvents := results.BeginBlockEvents
 		for _, txResults := range results.TxsResults {
 			tmEvents = append(tmEvents, txResults.Events...)
 		}
+		tmEvents = append(tmEvents, results.EndBlockEvents...)
```

Comment:
Looks unrelated, but might as well change this too, for consistency.
This covers all the occurrences of `EndBlockEvents` in the codebase.